src/eureka/S5_lightcurve_fitting/models/BatmanModels.py

```diff
@@ -121,7 +121,7 @@ def eval(self, channel=None, **kwargs):
 
             # Enforce physicality to avoid crashes from batman by returning
             # something that should be a horrible fit
-            if not ((0 < bm_params.rp) and (0 < bm_params.per) and
+            if not ((0 < bm_params.per) and
```

Comment:
Does batman actually allow negative radii without crashing? And if we're making this change, we should also make the same change in the src/eureka/S5_lightcurve_fitting/differentiable_models/StarryModel.py file (and make any tweaks that may be needed to allow negative radii in starry, e.g. just putting a negative sign on the lightcurve or something). The rp>0 prior is enforced here:
https://github.com/kevin218/Eureka/blob/3c1092661415f429c4ea685f898afa09679990d9/src/eureka/S5_lightcurve_fitting/differentiable_models/PyMC3Models.py#L353

Reply:
Batman does allow negative radii without crashing. I'm unsure if starry does, I've never used it.